instruments/keithley/keithley580.py

```diff
@@ -416,7 +417,8 @@ def measure(self):
         :rtype: `~quantities.quantity.Quantity`
         """
         self.trigger()
-        return self.parse_measurement(self.query(''))['resistance']
+        self.sendcmd('')
```

Comment:
Same as above, `sendcmd('')` used to send a ':'. 
`query` also removes the last character, which we now have to do manually.